main.go

```
@@ -0,0 +1,73 @@
+
+func init() {
+	// Hardcoded credentials - security issue
+	db, _ = sql.Open("mysql", "root:password123@tcp(127.0.0.1:3306)/testdb")
+}
+
+func main() {
+	http.HandleFunc("/search", searchHandler)
+	http.HandleFunc("/execute", executeHandler)
+
+	// Starting server without TLS
+	http.ListenAndServe(":8080", nil)
```

golangci-lint (1.62.2)

Error return value of `http.ListenAndServe` is not checked

(errcheck)